sdk/resourcemanager/azure-resourcemanager-redis/src/main/java/com/azure/resourcemanager/redis/models/RedisCache.java

```diff
@@ -27,52 +27,116 @@ public interface RedisCache extends GroupableResource<RedisManager, RedisResourc
     Updatable<RedisCache.Update>, SupportsListingPrivateLinkResource, SupportsListingPrivateEndpointConnection,
     SupportsUpdatingPrivateEndpointConnection {
 
-    /** @return exposes features available only to Premium Sku Redis Cache instances. */
+    /**
+     * Gets an instance of the Azure Redis cache with Premium SKU.
+     *
+     * @return exposes features available only to Premium Sku Redis Cache instances.
+     */
     RedisCachePremium asPremium();
 
-    /** @return returns true if current Redis Cache instance has Premium Sku. */
+    /**
+     * Checks whether current Redis Cache instance has Premium Sku.
+     *
+     * @return returns true if current Redis Cache instance has Premium Sku.
+     */
     boolean isPremium();
 
-    /** @return the provisioningState value */
+    /**
+     * Gets the provisioning state.
+     *
+     * @return the provisioningState value
+     */
     String provisioningState();
 
-    /** @return the hostname value */
+    /**
+     * Gets the host name.
+     *
+     * @return the hostname value
+     */
     String hostname();
 
-    /** @return the port value */
+    /**
+     * Gets the port.
+     *
+     * @return the port value
+     */
     int port();
 
-    /** @return the sslPort value */
+    /**
+     * Gets ssl port.
+     *
+     * @return the sslPort value
+     */
```

Comment:
SSL, uppercase

Reply:
Fixed by the [commit#83f5296](https://github.com/Azure/azure-sdk-for-java/pull/43632/commits/83f5296fc3d1a9f42d79e303f0f427c3a018c029)